src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -255,10 +255,14 @@ export default class ReimbursementRequestService {
 
     if (!oldReimbursementRequest) throw new NotFoundException('Reimbursement Request', requestId);
     if (oldReimbursementRequest.dateDeleted) throw new DeletedException('Reimbursement Request', requestId);
-    if (oldReimbursementRequest.recipientId !== submitter.userId)
-      throw new AccessDeniedException(
-        'You do not have access to delete this reimbursement request, only the creator can edit a reimbursement request'
-      );
+    try {
+      await validateUserIsPartOfFinanceTeam(submitter);
+    } catch {
+      if (oldReimbursementRequest.recipientId !== submitter.userId)
+        throw new AccessDeniedException(
+          'You do not have access to edit this reimbursement request, only the creator or finance team can edit a reimbursement request'
```

Comment:
Let's shrink the error message to say `Only the creator or finance team can edit a reimbursement request` since we know this error is the access denied type.